### PR TITLE
change logger to log module

### PR DESCRIPTION
**Pull request by watsy0007**

### CODE REVIEW

**Comment by watsy0007** [comment]
ok. I'll try it on the weekend.

**Comment by Acciaiodigitale** [comment]
Thank you @watsy0007!

File: src/scry/log.cr [hunk 1]
```diff
@@ -1,13 +1,19 @@
 require "log"
 require "./client"
+require "logger"
+require "log/entry"
 
 module Scry
   module Log
     class_property logger : ::Log = ::Log.for("")
```
Comment (bew) [inline src/scry/log.cr:8]: I don't think this is needed anymore, I'd suggest renaming the current `Scry::Log` to `Scry::Logging` (or even simply put the `ClientLogger` in `Scry` directly).

Then we can have `Scry::Log` be the logger instance (from `::Log.for("scry")`)

Comment (watsy0007) [reply src/scry/log.cr:8]: nice .

**Comment by watsy0007** [comment]
@bew I'm still trying to learn how to write it . so I close the issue first.

**Comment by bew** [comment]
No problem, feel free to re-open and continue iterating over your work

**Pull request closed without merging**